beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/aws/S3ClientTest.java

```diff
@@ -207,6 +207,25 @@ void deleteObjectsInDirectory() {
     assertThat(amazonS3.doesObjectExist(bucket, key2)).isFalse();
   }
 
+  @Test
+  void deleteThousandObjectsInDirectory() {
+    ArrayList<String> keys = new ArrayList<>();
+    int totalObjects = 1000;
+    for(int i = 1; i <= totalObjects; i++){
+      var key = keyRoot + "/file" + i;
+      keys.add(key);
+    }
+    keys.parallelStream().forEach(key -> amazonS3.putObject(bucket, key, content));
+    int numberOfObjectsLeft = amazonS3.listObjects(bucket, keyRoot).getObjectSummaries().size();
+    assertThat(numberOfObjectsLeft).isEqualTo(totalObjects);
+
+    List<String> result = s3Client.deleteObjects(bucket, keys);
```

Comment:
i saw the check for the number of keys deleted, but you should check that the names of the keys are correct too (not sure if the order matters too..)

Reply:
I don't think we need to check that since we are creating the files from the `keys` list, which is the same used for the deletion.

Reply:
the keys list returned from that method is created with logic that we wrote, so it should be verified too

Reply:
added assertion